config/locales/jobseekers.yml

```diff
@@ -838,6 +841,7 @@ en:
         subjects:
           one: Subject
           other: Subjects
+        training_and_cpd: Training and continuing personal development (CPD)
```

Comment:
I saw this string hardcoded in multiple views. Maybe we could point to this locale value instead?